apps/dapp-console/app/components/Selects/L2NetworkSelect.tsx

```diff
@@ -0,0 +1,82 @@
+import { SuperchainNetwork, superchain } from '@/app/constants/superchain'
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+} from '@eth-optimism/ui-components'
+import { useCallback, useMemo, useState } from 'react'
+import { Chain } from 'viem'
+import { optimism } from 'viem/chains'
+import { Network } from '@/app/components/Network'
+
+export type SuperchainDropdownProps = {
+  includeTestnets?: boolean
+  onNetworkChange?: (network: L2NetworkSelectItem) => void
+}
+
+export type L2NetworkSelectItem = {
+  id: number
+  chain: Chain
+  config: SuperchainNetwork
+}
+
+export const L2NetworkSelect = ({
+  includeTestnets,
+  onNetworkChange,
+}: SuperchainDropdownProps) => {
+  const [selectedNetwork, setSelectedNetwork] = useState<L2NetworkSelectItem>({
+    id: optimism.id,
+    chain: optimism,
+    config: superchain.op,
+  })
+
+  const allChains = useMemo(() => {
+    return Object.values(superchain).reduce((arr, config) => {
```

Comment:
should this be filtered to only include l2s? I think with the way it is written now l1 eth would be included in this.